services/xous-ticktimer/src/platform/precursor/implementation.rs

```diff
@@ -1,29 +1,47 @@
 const TICKS_PER_MS: u64 = 1;
 
+use core::sync::atomic::{AtomicU32, AtomicUsize, Ordering};
 use std::collections::BTreeMap;
 
-use crate::TimerRequest;
 use crate::TimeoutExpiry;
+use crate::TimerRequest;
 
 use susres::{RegManager, RegOrField, SuspendResume};
 use utralib::generated::*;
 
+use xous::definitions::MessageSender;
+
+/// Add some slack to ensure that when we set the timeout, it
+/// doesn't end up in the past due to processing latencies.
+const LATENCY_SLACK_MS: i64 = 3;
```

Comment:
This line seems to impact performance a bit.

trng outputs 15kB/s previously

at a setting of 3, we get about 6 kB/s.

reducing this to 1 gives us 10kB/s, and the system still seems stable running the trng test.

What do you think about adjusting this constant?

Reply:
Go ahead and adjust it downwards to 0, and add a note. Since the hardware uses `<=` it should be fine.

Reply:
Got it. I am doing a test run with this at 0 now. The speed of the TRNG is back up to 14.4Kb/s.

Reply:
This doesn't work, it makes the system unstable. I get a panic like this:

```
PANIC in PID 2:
thread 'main' panicked at 'assertion failed: ticktimer.last_response() == sender', services/xous-ticktimer/src/main.rs:197:21
 ```